CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main / unreleased
 
+* [FEATURE] Ingester: Add optional CPU/memory utilization based read limiting, considered experimental. Disabled by default, enable via `-ingester.utilization-based-limiting-enabled` flag. #5012
```

Comment:
```suggestion
* [FEATURE] Ingester: Add optional CPU/memory utilization based read-path limiting, considered experimental. Disabled by default, enable via `-ingester.utilization-based-limiting-enabled` flag. #5012
```

Reply:
I would also mention the other new config options.

Reply:
Done.